docs/installation.rst

```diff
@@ -37,46 +37,57 @@ Prerequisites
 
 Installation
 ^^^^^^^^^^^^
-
-Execute the below command CUDA 9.0 based build:
+To install a CUDA 9.0 based DALI build, execute the following command:
 
 .. code-block:: bash
 
-    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/9.0 nvidia-dali
+    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-cuda90
 
-Starting DALI 0.8.0 for CUDA 10.0 based build use:
+For CUDA 10.0 based build use:
 
 .. code-block:: bash
 
-   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/10.0 nvidia-dali
+   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-cuda100
 
 .. note::
 
-  Starting 0.6.1 the ``nvidia-dali`` package no longer contains prebuilt versions of the DALI TensorFlow plugin, so you need to install DALI TensorFlow plugin for the currently installed version of TensorFlow:
+  Since version 0.6.1, DALI doesn't contain prebuilt versions of the DALI TensorFlow plugin. It needs to be installed as a separate package which will be built against the currently installed version of TensorFlow:
 
 .. code-block:: bash
 
-    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/9.0 nvidia-dali-tf-plugin
-
-Starting DALI 0.8.0 for CUDA 10.0 based build execute:
+   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-tf-plugin-cuda90
 
 .. code-block:: bash
 
-   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/10.0 nvidia-dali-tf-plugin
-
+   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-tf-plugin-cuda100
 
-Installing this package will install ``nvidia-dali`` and its dependencies, if these dependencies are not already installed. The package ``tensorflow-gpu`` must be installed before attempting to install ``nvidia-dali-tf-plugin``.
+Installing this package will install ``nvidia-dali-cudaXXX`` and its dependencies, if these dependencies are not already installed. The package ``tensorflow-gpu`` must be installed before attempting to install ``nvidia-dali-tf-plugin-cudaXXX``.
 
 .. note::
 
-  The package ``nvidia-dali-tf-plugin`` has a strict requirement with ``nvidia-dali`` as its exact same version.
-  Thus, installing ``nvidia-dali-tf-plugin`` at its latest version will replace any older ``nvidia-dali`` versions already installed with the latest.
+  The package ``nvidia-dali-tf-plugin-cudaXXX`` has a strict requirement with ``nvidia-dali-cudaXXX`` as its exact same version.
+  Thus, installing ``nvidia-dali-tf-plugin-cudaXXX`` at its latest version will replace any older ``nvidia-dali-cudaXXX`` versions already installed with the latest.
```

Comment:
Can we paraphrase these two a little bit? Reading those makes my mind dry xD
```suggestion
  The packages ``nvidia-dali-tf-plugin-cudaXXX`` and ``nvidia-dali-cudaXXX`` shall be in exactly the same version.
  Therefore, installing the latest ``nvidia-dali-tf-plugin-cudaXXX``, will replace any older ``nvidia-dali-cudaXXX`` version already installed.
```

Reply:
Will do